ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/support/ArticleParser.java

```diff
@@ -0,0 +1,194 @@
+package org.cru.godtools.articles.aem.service.support;
+
+import org.cru.godtools.articles.aem.model.Article;
+import org.cru.godtools.articles.aem.model.Attachment;
+import org.json.JSONObject;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * This class handles parsing any AEM json calls into DOA objects.
+ */
+public class ArticleParser {
+
+    public static final String ARTICLE_LIST_KEY = "article_list_key";
+
+    public static final String ATTACHMENT_LIST_KEY = "attachment_list_key";
+
+    private static List<Attachment> attachmentList = new ArrayList<>();
+
+    private static List<Article> articleList = new ArrayList<>();
```

Comment:
these 2 lists are still going to cause problems. They aren't cleared when parsing a second article json object.